firestore/integration_test.go

```diff
@@ -651,6 +651,13 @@ func TestIntegration_QueryDocuments(t *testing.T) {
 	}{
 		{q, wants},
 		{q.Where("q", ">", 1), wants[2:]},
+		{q.Where("q", "<", 1), wants[:1]},
+		{q.Where("q", "==", 1), wants[1:2]},
+		{q.Where("q", "!=", 0), wants[1:]},
+		{q.Where("q", ">=", 1), wants[1:]},
+		{q.Where("q", "<=", 1), wants[:2]},
+		{q.Where("q", "in", [0, 1]), wants[:2]},
+		{q.Where("q", "not-in", [0, 1]), wants[2:]},
```

Comment:
Looks like you need to fully initialize the slice here. Something like:
```suggestion
		{q.Where("q", "in", []int{0, 1}), wants[:2]},
		{q.Where("q", "not-in", []int{0, 1}), wants[2:]},
```